service/create/service.go

```diff
@@ -163,14 +174,33 @@ func (s *Service) Boot() {
 						return
 					}
 
+					// Create AWS client
+					s.awsConfig.Region = cluster.Spec.AWS.Region
+					awsSession, ec2Client := awsutil.NewClient(s.awsConfig)
+
+					// Create KMS key
+					kmsSvc := kms.New(awsSession)
+					key, err := kmsSvc.CreateKey(&kms.CreateKeyInput{})
```

Comment:
What kind of key is this? Is this unique for each EC2 instance or K8S cluster? Should we attach some K8S cluster information so we know how the KMS key and the cluster is associated with each other?